recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set version = "0.6" %}
-{% set sha256 = "b4be2b3438971588dc306116658f475916755ad8fcc0618e79023846ce94f97b" %}
```

Comment:
Indeed. It is mainly to make it quick and easy to get to. Thanks for moving it back.